src/qililab/platform/platform.py

```diff
@@ -837,38 +844,56 @@ def _execute_qprogram_with_quantum_machines(  # pylint: disable=too-many-locals,
         threshold_rotations: dict[str, float | None] = {},
         thresholds: dict[str, float | None] = {},
         calibration: Calibration | None = None,
+        dataloss_tries: int = 3,
```

Comment:
why a different dataloss_tries (3) than before (2)?

Reply:
This is thee number of loops suggested by HW as too many may be too long on queue (they also use 5 though)